js/pages/settings.js

```diff
@@ -99,16 +99,22 @@ export const renderSettingsPage = async () => {
     formVisBools.isMailingAddressFormDisplayed = false;
     formVisBools.isLoginFormDisplayed = false;
     if (userData[cId.userProfileSubmittedAutogen] === cId.yes) {
+      let headerMessage = '';
+      if (!isParticipantDataDestroyed) {
+        if (userData[cId.verification] !== cId.verified) {
+            headerMessage = "Thank you for joining the National Cancer Institute's Connect for Cancer Prevention Study. Your involvement is very important. We are currently verifying your profile, which may take up to 3 business days.";
+        }
+      } else {
+        headerMessage = `We have deleted your information based on the data destruction request we received from you. If you have any questions, please contact the <a href="#support">Connect Support Center</a>.`;
+      }
+
```

Comment:
Nice refactor in displaying the header message for three different scenarios

Reply:
Thanks Gene!